nf_core/modules/lint/meta_yml.py

```diff
@@ -72,7 +72,7 @@ def meta_yml(module_lint_object: ComponentLint, module: NFCoreComponent, allow_m
     # Confirm that the meta.yml file is valid according to the JSON schema
     valid_meta_yml = False
     try:
-        with open(Path(module_lint_object.modules_repo.local_repo_dir, "modules/meta-schema.json")) as fh:
+        with open(Path("/Users/jmir/Work/modules", "modules/meta-schema.json")) as fh:
```

Comment:
wrong path

Reply:
oops thanks! I was testing and missed that I pushed it